_posts/en/meetings/2016-09-22-meeting.md

```diff
@@ -29,7 +29,7 @@ version: 1
 
 Bitcoin Core 0.13.0 is [released](/en/2016/08/23/release-0.13.0/) on 2016/08/23. The next point release 0.13.1 will probably include the segregated witness softfork activation logic, among other bugfixes and optimizations.
 
-In [last weeks meeting](/en/meetings/2016/09/15/#release) developers decided to drop PR [#8635][] (Enforce mandatory softfork flags for segwit block/tx) from the [0.13.1 milestone](https://github.com/bitcoin/bitcoin/milestone/22) tag.
+In [last weeks meeting](/en/meetings/2016/09/15/) developers decided to drop PR [#8635][] (Enforce mandatory softfork flags for segwit block/tx) from the [0.13.1 milestone](https://github.com/bitcoin/bitcoin/milestone/22) tag.

```

Comment:
This is not a dead link: https://bitcoincore.org/en/meetings/2016/09/15/#release

Reply:
That's odd. In the HTML I see `<h2 id="release">0.13.1 release</h2>`, but there's nothing in the [markdown](https://raw.githubusercontent.com/bitcoin-core/bitcoincore.org/master/_posts/en/meetings/2016-09-15-meeting.md). Where does that get added?

Reply:
I think an anchor based on the title is automatically added for the toc.

Reply:
@sjors the Markdown parser used on the site automatically generates anchors for any headers using the text of the header.  ID tags in HTML4 can't start with numbers or punctuation, so the "0.13.1 " is stripped and the generated id is just "release".

I'm guessing you included this in the PR because it broke when you upgraded the site.  I'm not sure exactly what happened, but it could be that the algorithm for generating the header links changed---in HTML5, id tags can begin with numbers, so possibly this anchor changed.  If you want, you can manually choose the anchor in the markdown so that it'll remain consistent through upgrades:

```markdown
## 0.13.1 release {#release}
```

Reply:
I indeed changed it because the linter started complaining when I tried upgrading.